src/Dflydev/FigCookies/SetCookies.php

```diff
@@ -86,23 +86,21 @@ public function renderIntoSetCookieHeader(ResponseInterface $response) : Respons
      * Create SetCookies from a collection of SetCookie header value strings.
      *
      * @param string[] $setCookieStrings
-     * @return static
      */
-    public static function fromSetCookieStrings(array $setCookieStrings) : self
+    public static function fromSetCookieStrings(array $setCookieStrings): self
     {
-        return new static(array_map(function (string $setCookieString) : SetCookie {
+        return new static(array_map(static function (string $setCookieString): SetCookie {
             return SetCookie::fromSetCookieString($setCookieString);
         }, $setCookieStrings));
     }
 
     /**
      * Create SetCookies from a Response.
-     *
      */
-    public static function fromResponse(ResponseInterface $response) : SetCookies
+    public static function fromResponse(ResponseInterface $response): SetCookies
     {
-        return new static(array_map(function (string $setCookieString) : SetCookie {
+        return new static(array_map(static function (string $setCookieString): SetCookie {
             return SetCookie::fromSetCookieString($setCookieString);
-        }, $response->getHeader(static::SET_COOKIE_HEADER)));
+        }, $response->getHeader(self::SET_COOKIE_HEADER)));
```

Comment:
changes are low (http spec), but also this constant could be overridden

Reply:
I think I've completely underestimated the `static::` vs `self::` implications in a few places... Is doing `static::` bad here for some reason?